docs/framework/configure-apps/file-schema/wcf/httpstransport.md

```diff
@@ -24,12 +24,11 @@ Specifies an HTTP transport for transmitting SOAP messages for a custom binding.
                 maxBufferSize="Integer"
                 maxReceivedMessageSize="Integer"
                 proxyAddress="Uri"
-                proxyAuthenticationScheme="None/Digest/Negotiate/Ntlm/Basic/Anonymous"
+                proxyAuthenticationScheme="None/Digest/Negotiate/Ntlm/Basic/Anonymous/IntegratedWindowsAuthentication"
```

Comment:
Commenting again as previous conversation was marked as resolved.

I'm pretty certain this is incorrect. If you look at [this code](https://referencesource.microsoft.com/#System.ServiceModel/System/ServiceModel/HttpProxyCredentialType.cs,29) you can see that it will throw an exception if anything other than Anonymous, Basic, Digest, Ntlm or Negotiate is specified.

Reply:
Is the table also incorrect?

Reply:
I believe so. I'll need to write some test code to verify in case I've missed something. I'll update this conversation once I have done so.

Reply:
I just verified and we throw an exception if `IntegratedWindowsAuthentication` is specified. This is because it's value is actually `Negotiate | Ntlm` and we only allow a single authentication scheme to be specified. So it does need to be removed from the table. It might be worth adding a note after the where it says this is of type AuthenticationSchemes that this value isn't supported.